### ISSUE TITLE
esriClassifyNaturalBreaks with break count issue

### ISSUE THREAD

**Issue by mtrcn** [koopjs/koop#1019]
https://github.com/koopjs/koop/blob/61cd8d57bead390de03c8840d47c35ff2d418f05/packages/winnow/src/calculate-class-breaks/index.js#L15

I think I found a bug on this line because if I provide `breakCount = 10` for a value array with 6 elements, it changes breakCount to 6 as the given break count number is higher than the array size but `classybrew` fails at this. 

Check this code pen; https://codepen.io/mtrcn/pen/zYQKYqg

It gives `Cannot read properties of undefined (reading '4')` error in the console which I also get the same in KoopJS. If I set `brew.setNumClasses(5);`, it works. 

My `classificationDef` parameter is;

```json
{"type":"classBreaksDef","breakCount":"10","classificationField":"easting","classificationMethod":"esriClassifyNaturalBreaks"}
```

**Comment by rgwozdz** [comment]
@mtrcn - thanks for reporting this. We'll look into it ASAP.

**Comment by mtrcn** [comment]
@rgwozdz thank you, let me know if you need anything else to support you in fixing the issue.

**Comment by rgwozdz** [comment]
@mtrcn - this should be fixed in just released 10.4.13.

**Comment by mtrcn** [comment]
@rgwozdz thank you so much for quick response on this matter.
